src/components/templates/Audience/Audience.scss

```diff
@@ -41,6 +41,8 @@ $spacing: calc(1 * min(var(--seat-size), var(--seat-size-min)));
       flex-direction: column;
       align-items: center;
 
+      pointer-events: all;
```

Comment:
- https://css-tricks.com/almanac/properties/p/pointer-events/
  - While the pointer-events property takes eleven possible values, all but three of them are reserved for use with SVG. The three valid values for any HTMl element are:
    - `none`: prevents all click, state and cursor options on the specified HTML element
    - `auto`: restores the default functionality (useful for use on child elements of an element with `pointer-events: none;` specified
    - `inherit` will use the pointer-events value of the element’s parent
    
---

Based on the above, it looks like we want to use `pointer-events: auto;` here, rather than `all`

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/578 to ensure this isn't missed/lost.